packages/nimble-components/src/table/template.ts

```diff
@@ -28,13 +28,14 @@ const isTableColumn = (): ElementsFilter => {
 export const template = html<Table>`
     <template role="table">
         <div class="table-container">
-            <div role="rowgroup" class="header-container" style="margin-right: ${x => x.virtualizer.headerContainerMarginRight}px;">
+            <div role="rowgroup" class="header-container">
                 <div class="header-row" role="row">
                     ${repeat(x => x.columns, html<TableColumn>`
                         <${DesignSystem.tagFor(TableHeader)} class="header">
                             ${x => x.textContent}
                         </${DesignSystem.tagFor(TableHeader)}>
                     `)}
+                    <div style="width: ${x => x.virtualizer.headerContainerMarginRight}px;"></div>
```

Comment:
Maybe add a template comment describing why we need it (i.e. one of those `${'' /* Spacer div to extend the header space over the scrollbar */}`) or a descriptive class like `header-scrollbar-spacer` or something.

Reply:
Added the CSS class.